client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -53,9 +54,7 @@
   import Counters from '@/components/Counters.vue';
   import MultiLinePlot from '@/components/widgets/charts/MultiLinePlot.vue';
 
-  import { donuSimulateToD3 } from '@/utils/DonuUtil';
-
-  import * as HMI from '@/types/types';
```

Comment:
should we stop using this style of import in the future?

Reply:
hmmm good question, i think i made this change during a merge and merged the old style. ive got nothing against this style or the specific style of import personally.

Reply:
_Parfait_, because I prefer the `* as HMI` way, it makes it easier to understand what we are doing in the code, and to see where the type is coming from.
Specially that with tree-shaking there is no memory issue now to do so.